modules/nw-operator-cr.adoc

```diff
@@ -372,7 +352,7 @@ defaultNetwork:
 
 [discrete]
 [id="nw-operator-cr-kubeproxyconfig_{context}"]
-=== kubeProxyConfig object configuration
+=== kubeProxyConfig object configuration (OpenShiftSDN container network interface only)
```

Comment:
It can be configured for some third-party network plugins as well, but I don't think we discuss third-party network plugins anywhere in the official docs (as opposed to, eg, access.redhat.com) so I guess this is correct.